web/src/components/TaskPageSwitchComponent.tsx

```diff
@@ -62,6 +62,9 @@ export const TaskPageSwitchComponent = ({
     "search-business-name": (
       <BusinessFormation task={getTaskFromRoadmap(roadmap, task.id)} displayContent={displayContent} />
     ),
+    "public-accountant-license": (
+      <LicenseTask task={task} CMS_ONLY_disable_overlay={CMS_ONLY_disable_overlay} />
+    ),
```

Comment:
[dust] I might have put this up with the other `LicenseTask`s just so things are grouped together.

Reply:
I started to do this and then changed course. I should have gone with my first mind! I'll change it. Thanks, JP.